src/library_manager/lib_manager.c

```diff
@@ -395,22 +395,21 @@ uint32_t lib_manager_allocate_module(const struct comp_driver *drv,
 	return 0;
 }
 
-int lib_manager_free_module(const struct comp_driver *drv,
-			    struct comp_ipc_config *ipc_config)
+int lib_manager_free_module(const uint32_t component_id)
 {
 	struct sof_man_fw_desc *desc;
 	struct sof_man_module *mod;
-	uint32_t module_id = IPC4_MOD_ID(ipc_config->id);
+	const uint32_t module_id = IPC4_MOD_ID(component_id);
 	uint32_t entry_index = LIB_MANAGER_GET_MODULE_INDEX(module_id);
 	int ret;
 
-	tr_dbg(&lib_manager_tr, "lib_manager_free_module(): mod_id: %#x", ipc_config->id);
+	tr_dbg(&lib_manager_tr, "lib_manager_free_module(): mod_id: %#x", component_id);
```

Comment:
same here: "module ID" seems to be a better fit

Reply:
@lyakh: The field `id` in the structure `comp_ipc_config` is described as "component id". So I think this name better illustrates where this value comes from.